pyproject.toml

```diff
@@ -3,22 +3,22 @@ requires = [
     "setuptools>=59.6",
     "pybind11>=2.10.0",
     "cmake>=3.22",
-    "numpy>=1.21",
+    "numpy==1.25", # this is important to keep fixed. It also means anyone using something other than 1.22 won't be able to use this library
     "wheel",
     "ninja"
 ]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "diskannpy"
-version = "0.5.0.rc2"
+version = "0.5.0.rc4"
```

Comment:
we tagged 0.5.0.rc3 in github even though our pyproject.toml wasn't updated.  manually keeping this stuff in sync is a huge dang pain.